### DISCUSSION TITLE
Does the Navidrome search consider ID3 V2.4 Tags like TMOO (Mood)?

### DISCUSSION THREAD

**Discussion by FlixCR** [navidrome/navidrome#2651] [Q&A]
I´m collecting a special music catalog that would need a special ID3 tagging sheme. For example I need "Moods", special Genres and more. I could define them and write them in my MP3s ID3 Metatags. But does the navidrome search take all (also custom created) ID3 tags into account?

Thanks! Felix

**Comment by certuna** [comment]
TMOO is not read/stored by Navidrome at all, at the moment.

With the recent addition of this tag to the OpenSubsonic API it's pretty likely that ND will eventually get it, but it may take a while.
